app/scripts/modules/core/src/cluster/cluster.service.ts

```diff
@@ -31,7 +32,7 @@ export class ClusterService {
       const serverGroupLoader = API.one('applications')
         .one(application.name)
         .all('serverGroups');
-      dataSource.fetchOnDemand = clusters.length > ClusterService.ON_DEMAND_THRESHOLD;
```

Comment:
hmm, I guess we should delete this `ON_DEMAND_THRESHOLD` const from the class?

Reply:
how embarrassing